specforge/trainer_helper/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .optimizer import BF16Optimizer
+from .tracker import Tracker, build_tracker
+
+__all__ = ["BF16Optimizer", "Tracker", "build_tracker"]
```

Comment:
I think this module name is a bit weird, `helper` usually refers to some utilities functions, but this module contains components necessary for training.  I guess these can be an independent file, i.e. `specforge.tracker` and `specforge.optimizer`.